doc/plugin_server_nodeattestor_jointoken.md

```diff
@@ -0,0 +1,9 @@
+# Server plugin: nodeattestor-jointoken
+
+The `nodeattestor-jointoken` plugin attests a node based on a pre-shared, one-time-use join token.
```

Comment:
Worth describing the relationship between this and the agent version of this plugin. Must both be enabled for join tokens on nodes to work?

Reply:
LGTM